DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/ManageCourseController.cs

```diff
@@ -43,6 +46,17 @@ public IActionResult Index(int customisationId)
             return View(model);
         }
 
+        [HttpGet]
+        [Route("LearningPathwayDefaults/New")]
+        public IActionResult EditLearningPathwayDefaultsNew(int customisationId)
+        {
+            var model = new EditLearningPathwayDefaultsViewModel();
+
+            SetEditLearningPathwayDetailsTempData(model);
+
+            return RedirectToAction(nameof(EditLearningPathwayDefaults), new { customisationId });

```

Comment:
I don't really see the benefit of using nameof here when the rest of the file is full of strings for view names/actions. Either we need to start using this everywhere we can, or not bother.